src/lib/ddcci.c

```diff
@@ -1100,6 +1103,10 @@ int ddcci_close(struct monitor* mon)
 			free(mon->caps.vcp[i]);
 		}
 	}
+
+	if(mon->caps.raw_caps) {
+		free(mon->caps.raw_caps);
+	}
```

Comment:
free does not mind NULL parameter, so you don't need the if block.

Reply:
True. Thanks for reviewing, feel free to do complete review.

I'll be leaving to work soon, and today evening I'm travelling on a short vacation. I'll be back in couple of days, then I'll correct points mentioned in review.

I intentionally left PR open, wondering whether, or hoping, that someone would do a review.